docs/deployment/linux/ubuntu.txt

```diff
@@ -47,17 +47,12 @@ the CrateDB repository.
 
 .. _Apt: https://wiki.debian.org/Apt
 
-Download the CrateDB GPG key:
+Download and import the CrateDB GPG key:
 
 .. code-block:: sh
 
-   sh$ wget https://cdn.crate.io/downloads/apt/DEB-GPG-KEY-crate
+   sh$ wget -qO - https://cdn.crate.io/downloads/deb/DEB-GPG-KEY-crate | apt-key add -
```

Comment:
Adding the apt key requires su privileges, so it needs to be `... | sudo apt-key add -`

Reply:
Roger, I will update this tomorrow